data/map.txt

```diff
@@ -27846,7 +27846,7 @@ planet Janiculum
 
 planet Kaeyin
 	attributes uninhabited
-	landscape land/myrabella5
+	landscape land/city19
```

Comment:
When I said this isn't a station I didn't mean just change the image name to something other than station. Planets should have proper landscape images while stations have interior images, but this is a planet with an interior image.